modules/lang-painless/src/main/java/org/elasticsearch/painless/Def.java

```diff
@@ -72,14 +72,24 @@
 
         private static final MethodHandle OBJECT_ARRAY_MH = ARRAY_TYPE_MH_MAPPING.get(Object[].class);
 
+        // NOTE: the following are actually used, javac just does not know :)
+        @SuppressWarnings("unused")
```

Comment:
@uschindler That's awesome about the missing getters.  Thanks for making that happen!